cli/plasmo/src/features/manifest-factory/base.ts

```diff
@@ -419,17 +420,47 @@ export abstract class PlasmoManifest<T extends ExtensionManifest = any> {
       .then((files) =>
         Promise.all(
           files
-            .filter((f) => f.isFile())
+            .filter((f) =>
+              f.isFile() && filterFile ? filterFile(f.name) : true
```

Comment:
could also have a default `() => true` arg value if this logic feels a bit messy